x/autopilot/module_ibc.go

```diff
@@ -17,7 +17,10 @@ import (
 	ibcexported "github.com/cosmos/ibc-go/v7/modules/core/exported"
 )
 
-const MaxMemoCharLength = 2000
+const (
+	MaxMemoCharLength     = 4096
```

Comment:
just noting this doesn't need to be a power of 2

Reply:
yeah I just borrowed it from the other PR. I can drop to 4000

Reply:
https://github.com/Stride-Labs/stride/pull/1273/commits/4611510526d160c4116fee48e08802ce7ed670c3

Reply:
Either works, just noting in case we look at this param in the future and wonder